salt/modules/deb_postgres.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide Postgres compatibility to salt for debian family specific tools.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import pipes
+
+# Import salt libs
+import salt.utils
+
+# Import 3rd-party libs
+import salt.ext.six as six
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'postgres'
```

Comment:
Indeed, modules are merged, something i did not known about the loader.
So yes, as there are no function no conflict, there is no issue left here.